internal/injector/builtin/yaml/databases/mongo.yml

```diff
@@ -0,0 +1,25 @@
+# Unless explicitly stated otherwise all files in this repository are licensed
+# under the Apache License Version 2.0.
+# This product includes software developed at Datadog (https://www.datadoghq.com/).
+# Copyright 2023-present Datadog, Inc.
+%YAML 1.1
+---
+meta:
+  name: go.mongodb.org/mongo-driver/mongo
+  description: Official Golang driver for MongoDB.
+  icon: database
+aspects:
+  - id: Inject mongotrace.NewMonitor
+    join-point:
+      one-of:
+        - function-call: go.mongodb.org/mongo-driver/mongo.Connect
+    advice:
+      - wrap-expression:
+          imports:
+            redis: go.mongodb.org/mongo-driver/mongo
+            mongotrace: gopkg.in/DataDog/dd-trace-go.v1/contrib/go.mongodb.org/mongo-driver/mongo
+          template: |-
+            func() (client *mongo.Client, err error) {
+              opts.Monitor = mongotrace.NewMonitor()
+              return mongo.Connect(ctx, opts)
```

Comment:
You need to have `mongo` defined in the `imports` map (it needs to bind all package names to the corresponding import path, even if the context implies this package is already imported, because this snippet is parsed in a separate context before being moved into the final updated AST)

Reply:
@RomainMuller You are right. I left a redis prefix from the example I based this PR.